source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,76 @@
+
+.. image:: images/robot-battery/boiled.jpg
+   :alt: An FRC battery that was boile by a high current dumb charger.
+
+Most modern chargers can charge the battery indefinitely without additional harm.  The picture above is from a high current dumb charger that boiled the acid until it produced steam and high pressure to open the safety vents.  As such a smart charger is highly recommended.
+
+Great care should be taken when handling batteries.  Dropping a battery may compromise its structural integrity and can lead to a bulge or crack in the battery case.  These batteries should be safely disposed of as soon as possible.  All retailers that sell batteries like car batteries should be able to dispose of it for you, some may charge a small fee.
+
+Batteries need to be kept away from both extreme heat and cold.  This generally means storing the batteries in a climate controlled area.
+
+Battery Load Tester
+^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/robot-battery/load-tester.jpg
+   :alt: Output screen of a common load tester showing properties of the battery.
+
+A battery load tester can be used as a quick way to determine the readiness of a battery.  It may provide information like: open-load voltage, voltage under load, internal resistance, and state of charge.  These metrics can be used to quickly confirm that a battery is ready for a match and even possibly help to identify some long term problems with the battery.
+
+While these batteries are called 12 volt batteries a fully charged battery should read around 13 volts.  Ideal internal resistance should be less than 0.015 Ohms.  The manufacturer specification for most batteries is 0.011 Ohms.  If a battery gets higher than 0.02 Ohms it is a good idea to consider not using that battery for competition matches.
+
+Battery Analyzer
+^^^^^^^^^^^^^^^^
+
+.. image:: images/robot-battery/cba-graph.jpg
+   :alt: Graph from a common battery analyzer plotting volts and AmpHrs for many different batteries.
+
+A battery analyzer is used to give a detailed inspection of battery performance.  It will provide graphs of battery performance over time.  This test takes significant time (roughly two hours) so it is less suited to testing during competition.  It is recommended to run this test on each battery every year to monitor and track its performance.  This will determine how it should be used: matches, practice, testing, or disposed of.
+
+Experts agree that at the standard 7.5 amps of load current competition batteries should have at least a 11.5 amp hour rating.  Anything less than that should only be used for practice or other less demanding use cases.
+
+Battery Best Practices
+^^^^^^^^^^^^^^^^^^^^^^
+
+* Teams should consider purchasing several new batteries each year to help keep their batteries fresh.  Elimination matches can require many batteries and there may not be enough time to recharge.
```

Comment:
I know ``*`` is valid syntax for bullets, but could it be replaced with ``-`` for consistency sake.